### DISCUSSION TITLE
[Create Next App] Example project expected to have lock files or not to?

### DISCUSSION THREAD

**Discussion by janpio** [vercel/next.js#36677] [Help]
Hey,

we have an example project that is installed via Create Next App and is also used in an official guide: https://vercel.com/guides/nextjs-prisma-postgres We just realized we are including both a `package-lock.json` and `yarn.lock` in that project, and it is causing some trouble with our own tooling of Prisma CLI (which assues by the presence of `yarn.lock` that Yarn is installed, which it might not be...). This realization lead to the question: 

Are example projects to be used with Create Next App _expected to have lock files_ or expected to _not_ have them? 

I see depending on the local setup and options it installs with Npm, Yarn or Pnpm - so maybe it is actually better _not_ to include one?

Best
Jan @ Prisma

**Comment by GabenGar** [comment]
Within a single project you have to stick to a single lock file and therefore decide if it uses npm/yarn/pnpm.
